components/constraints.js

```diff
@@ -12,8 +12,11 @@ class Constraints extends Component {
     if (!constraints) return <div />;
     return (
       <ul style={{ listStyle: 'none', paddingLeft: 0 }}>
-        {(constraints.get('default') || constraints.get('default') === 0) &&
-          <li>default value: {constraints.get('default')}</li>
+        {constraints.has('default') &&
+          (constraints.get('type') === 'boolean' ?
+            (constraints.get('default') ? 'default value: true' : 'default value: false') :
```

Comment:
- First it checks if schema's "default" property exists.
- If it does, it checks if the schema's "type" property is boolean.
  - If it is boolean, it checks if the value of schema's "default" property is true.
    - if yes, it displays: `default value: true`
    - if no, it displays: `default value: false`
  - If it isn't boolean, it just displays the value as `default value: constraints.get('default')`

**Examples:**

```json
"in_stock": {
  "type": "boolean",
  "description": "Whether the product is in stock or not",
  "example": true,
  "default": false
},
```

displays:

```
default value: false
```

```json
"in_stock": {
  "type": "boolean",
  "description": "Whether the product is in stock or not",
  "example": true,
  "default": true
},
```

displays:

```
default value: true
```

```json
"in_stock": {
  "type": "boolean",
  "description": "Whether the product is in stock or not",
  "example": true
},
```

displays:

```
// nothing
```

```json
"in_stock": {
  "type": "number",
  "description": "Whether the product is in stock or not",
  "example": true,
  "default": 0
},
```

displays:

```
default value: 0
```